corehq/tabs/tabclasses.py

```diff
@@ -1838,7 +1838,7 @@ def sidebar_items(self):
         if self.couch_user.is_superuser:
             from corehq.apps.enterprise.models import EnterprisePermissions
             if toggles.DOMAIN_PERMISSIONS_MIRROR.enabled_for_request(self._request) \
-                    or EnterprisePermissions.get_by_domain(self.domain).is_enabled:
+                    or EnterprisePermissions.is_source_domain(self.domain):
```

Comment:
This is a slight behavior change. This condition is a fallback so that the UI for configuring enterprise permissions will be visible in HQ's navigation if a configuration exists, even if the related feature flag has been turned off. 

Previously, the UI would show for all domains in any account with an active config, even without the flag on. Now, it will show only for the source domain. I think this is a minor enough change to be fine, especially considering this UI is superuser only.